### ISSUE TITLE
[Tool List] Page needs headings

### ISSUE THREAD

**Issue by MicheleAWilliams-A11y** [w3c/wai-evaluation-tools-list#65]
The page is missing subheadings to describe the content. This is necessary for WCAG (SC 2.4.6), and I suggest using a design similar to the Course List: https://github.com/w3c/wai-course-list/issues/279#issuecomment-1196420125

**Comment by VeraLange** [comment]
We created subheadings for the content sections like the course list has.
